astroplan/plots/mplstyles.py

```diff
@@ -0,0 +1,92 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from astropy.visualization import astropy_mpl_style
+
+__all__ = ['light_style_sheet', 'dark_style_sheet',
+           'available_style_sheets']
+
+available_styles = ["light_style_sheet", "dark_style_sheet"]
+
+def available_style_sheets():
+    """
+    Matplotlib style sheets available within astroplan.
+    """
+    return available_styles
+
+# One update specifically for astroplan.plots: raise bottom up from default 0.1
+astropy_mpl_style.update({'figure.subplot.bottom': 0.15})
```

Comment:
Good catch. I'm `deepcopy`ing the dict to make sure I don't overwrite astropy's.